core/modules/utils/dom/dom.js

```diff
@@ -204,6 +204,77 @@ exports.addEventListeners = function(domNode,events) {
 	});
 };
 
+/*
+Listen for resize events on a give domNode and execute a callback on resize
+http://www.backalleycoder.com/2013/03/18/cross-browser-event-based-element-resize-detection/
+*/
+exports.addResizeListener = function(domNode,callback) {
+	if(domNode && !domNode.__resizeListeners__) {
+		domNode.__resizeListeners__ = [];
```

Comment:
Strange variable names: `__asdf__`. IMO we shouldn't change variable name conventions in the middle of a single module.

Reply:
that naming is just for marking it and I just followed the naming from the example, see the link
do we have a naming convention for something like that? I don't know how to describe it - is it a not-browser-specific identifier that should look like one (..) ?

or should I just name it domNode.resizeListeners?

Reply:
I don't know. @Jermolene ??

Reply:
This is a custom property on a DOM node. I was trained not to use them 15 years ago with TWC because they used to generate memory leaks. I'm still not keen on the technique, to be honest.

Reply:
@Jermolene, I closed this PR because I found the technique was erratic ... there are probably better ways